docs/sources/next/javascript-api/k6-experimental/browser/element-handle/dblclick.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'dblclick([, options])'
+excerpt: 'Browser module: elementHandle.dblclick([, options]) method'
+---
+
+# dblclick([, options])
+
+Mouse double clicks the element.
```

Comment:
WDYT?

```suggestion
Mouse double clicks on the element.
```

Reply:
Resolved in 4dfadddfea4194c7656d8570c63b9089f35cf861